tools/windows_portable/build_portable.ps1

```
@@ -923,6 +960,58 @@ function Copy-Licenses([string]$BundleRoot, [pscustomobject[]]$Artifacts) {
 
+function Copy-RequiredQtLicenseDirectories([string]$BundleRoot) {
+  $sitePackages = Join-Path $BundleRoot "app\\site-packages"
+  $licensesDir = Join-Path $BundleRoot "licenses"
+  $required = @(
+    @{ Pattern = "pyqt6-*.dist-info"; Destination = "PyQt6" },
+    @{ Pattern = "pyqt6_qt6-*.dist-info"; Destination = "Qt" },
+    @{ Pattern = "pyqt6_sip-*.dist-info"; Destination = "PyQt6-sip" }
+  )
+
+  foreach ($entry in $required) {
+    $matches = @(Get-ChildItem -LiteralPath $sitePackages -Directory -Filter $entry.Pattern)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)